packages/live-status-gateway/api/schemas/adLibs.yaml

```diff
@@ -60,17 +60,64 @@ $defs:
               type: string
           required: [name, label]
           additionalProperties: false
-      tags:
-        description: Tags attached to this AdLib
-        type: array
-        items:
-          type: string
       publicData:
         description: Optional arbitrary data
     required: [id, name, sourceLayer, actionType]
     additionalProperties: false
+  adLib:
+    allOf:
+      - $ref: '#/$defs/baseAdLib'
+      - type: object
+        properties:
+          partId:
+            description: Unique id of the part which owns this AdLib
+            type: string
+    examples:
+      - id: 'C6K_yIMuGFUk8X_L9A9_jRT6aq4_'
+        partId: 'H5CBGYjThrMSmaYvRaa5FVKJIzk_'
+        name: Music video clip
+        sourceLayer: Video Clip
+        actionType:
+          - name: pvw
+            label: Preview
+        publicData:
+          fileName: MV000123.mxf
+  globalAdLib:
+    allOf:
+      - $ref: '#/$defs/baseAdLib'
+      - type: object
+        properties:
+          tags:
+            description: Tags attached to this AdLib
+            type: array
+            items:
+              type: string
```

Comment:
Why are tags now only available on `globalAdLib`'s? Aren't they available on all types of AdLibs?

Reply:
Ah, it's because it's in `IBlueprintPieceGeneric` as opposed to `IBlueprintAdLibPiece` so I didn't spot it, will fix.